src/aavrutiDJun20/MaxLengthOfName.java

```diff
@@ -0,0 +1,47 @@
+//Find Max length of a given names using array and Scanner
+
+package aavrutiDJun20;
+
+import java.util.Scanner;
+
+public class MaxLengthOfName {
+	
+	void maxLength(String[] listOfNames) {
+		int max = 0;
+		String maxLengthName = "";
+		for(int index=0;index<listOfNames.length;index++) {
+			if(listOfNames[index].length()>max) {
```

Comment:
you can remove variable max, and can use maxLengthName.length().
listOfNames[index].length()>maxLengthName.length()